nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/ListHDFS.java

```diff
@@ -176,7 +176,7 @@ private HDFSListing deserialize(final String serializedState) throws JsonParseEx
 
         // Build a sorted map to determine the latest possible entries
         for (final FileStatus status : statuses) {
-            if (status.getPath().getName().endsWith("_COPYING_")) {
+            if (status.getPath().getName().endsWith("_COPYING_") || status.getPath().getName().startsWith(".")) {
```

Comment:
I think we should try to be consistent with how ListFile works. It has a property to apply a regex to filer filenames and the default value is anything that doesn't start with a dot:

```
public static final PropertyDescriptor FILE_FILTER = new PropertyDescriptor.Builder()
            .name("File Filter")
            .description("Only files whose names match the given regular expression will be picked up")
            .required(true)
            .defaultValue("[^\\.].*")
            .addValidator(StandardValidators.REGULAR_EXPRESSION_VALIDATOR)
            .build();
```
This way the user can determine if they want dot files or not.